src/App.js

```diff
@@ -1,22 +1,34 @@
-import React from "react";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import { Routes, Route, useNavigate } from "react-router-dom";
 import "./App.css";
 import { Home, Review, Search } from "./Pages";
 import { Header } from "./Components";
 
 const App = () => {
+  const navigate = useNavigate();
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 800);
+  }, [navigate]);
```

Comment:
I'm not sure adding a 800ms forced loading message is the way to go.  Loading should be happening when we are waiting for responses from the api.   Let's remove it for now